gistUploader.js

```diff
@@ -1,11 +1,20 @@
-
 const axios = require('axios');
+const fs = require('fs');
+
+const API_URL = 'https://api.github.com/gists';
+const fileName = process.argv[3];
+const post = {};
```

Comment:
It seems this variable doesn't need to be defined outside of function `processFile`

Reply:
done